src/assemble_workflow/bundle_deb.py

```diff
@@ -0,0 +1,105 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import glob
+import shutil
+import subprocess
+
+from manifests.build_manifest import BuildManifest
+from system.os import deb_architecture
+
+
+class BundleDeb:
```

Comment:
Is this almost the same as `BundleRpm`? If so create a base class called maybe `BundleLinuxDistro` and subclass `BundleDeb` and `BundleRpm` from there. There's probably other opportunities to avoid copy-pasting code.

Reply:
Followed your recommendation here too. Please have a look again.